### PR TITLE
Clean Runtime Code

### PR DESCRIPTION
**Pull request by AurevoirXavier**

### CODE REVIEW

File: node/runtime/pangolin/src/pallets/bridge_messages.rs [hunk 1]
```diff
@@ -1,58 +1,47 @@
...
 	FeeMarketMessageAcceptedHandler, FeeMarketMessageConfirmedHandler, FeeMarketPayment,
 };
 use darwinia_support::evm::{ConcatConverter, IntoAccountId, IntoH160};
+use pallet_bridge_messages::Config;
 
 frame_support::parameter_types! {
 	pub const MaxMessagesToPruneAtOnce: MessageNonce = 8;
 	pub const MaxUnrewardedRelayerEntriesAtInboundLane: MessageNonce =
-		MAX_UNREWARDED_RELAYERS_IN_CONFIRMATION_TX;
+		bp_pangoro::MAX_UNREWARDED_RELAYERS_IN_CONFIRMATION_TX;
```
Comment (AsceticBear) [inline node/runtime/pangolin/src/pallets/bridge_messages.rs:16]: This should be `bp_panolin`?

File: node/runtime/pangolin/src/pallets/bridge_messages.rs [hunk 2]
```diff
@@ -1,58 +1,47 @@
...
 	pub const MaxUnconfirmedMessagesAtInboundLane: MessageNonce =
-		MAX_UNCONFIRMED_MESSAGES_IN_CONFIRMATION_TX;
+		bp_pangoro::MAX_UNCONFIRMED_MESSAGES_IN_CONFIRMATION_TX;
```
Comment (AsceticBear) [inline node/runtime/pangolin/src/pallets/bridge_messages.rs:18]: And this one.

File: node/runtime/pangoro/src/pallets/bridge_messages.rs [hunk 3]
```diff
@@ -1,56 +1,47 @@
...
 	type OutboundMessageFee = Balance;
 
-	type InboundPayload = FromPangolinMessagePayload;
-	type InboundMessageFee = pangolin_primitives::Balance;
-	type InboundRelayer = pangolin_primitives::AccountId;
+	type InboundPayload = bm_pangolin::FromPangolinMessagePayload;
+	type InboundMessageFee = bp_pangolin::Balance;
+	type InboundRelayer = bp_pangolin::AccountId;
 
-	type AccountIdConverter = AccountIdConverter;
```
Comment (AsceticBear) [inline node/runtime/pangoro/src/pallets/bridge_messages.rs:50]: The original `type AccountIdConverter = bp_pangolin::AccountIdConverter;` seems wrong.

File: node/runtime/pangolin/src/pallets/bridge_messages.rs (hunk 1 above)
Comment (AurevoirXavier) [reply node/runtime/pangolin/src/pallets/bridge_messages.rs:16]: https://github.com/darwinia-network/darwinia-bridges-substrate/blob/main/bin/rialto/runtime/src/lib.rs#L423

File: node/runtime/pangolin/src/pallets/bridge_messages.rs (hunk 2 above)
Comment (AurevoirXavier) [reply node/runtime/pangolin/src/pallets/bridge_messages.rs:18]: Same

File: node/runtime/pangoro/src/pallets/bridge_messages.rs (hunk 3 above)
Comment (AurevoirXavier) [reply node/runtime/pangoro/src/pallets/bridge_messages.rs:50]: We made a mistake before.

**Review by AsceticBear — APPROVED** [review]

**Pull request merged**